service/controller/v8/version_bundle.go

```diff
@@ -27,6 +27,11 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Update to 3.3.13. More info here: https://github.com/etcd-io/etcd/blob/master/CHANGELOG-3.3.md#v3313-2019-05-02",
 				Kind:        versionbundle.KindChanged,
 			},
+			{
+				Component:   "ignition",
+				Description: "Add name label for default and kube-system namespaces.",
+				Kind:        versionbundle.KindAdded,
+			},
```

Comment:
Isn't v8 already release?

Reply:
`v8` is `2.4.0` and it is WIP

Reply:
So why do we have v9?

Reply:
That's a question to @mazzy89 https://github.com/giantswarm/azure-operator/pull/525